app/services/host_aspects.rb

```diff
@@ -0,0 +1,93 @@
+module HostAspects
+  class Configuration
```

Comment:
It's kind of monolithic. If there was an option to create private classes - I would probably do it.
The main class here is the `Configuration` class, the 'Entry` is internal only, and the module just adds a singleton instance of the configuration.